libethereum/ValidationSchemes.h

```diff
@@ -0,0 +1,82 @@
+/*
+    This file is part of cpp-ethereum.
+
+    cpp-ethereum is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    cpp-ethereum is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with cpp-ethereum.  If not, see <http://www.gnu.org/licenses/>.
+*/
+/** @file ValidationSchemes.h
+ * @date 2018
+ */
+
+#include <string>
+#include <json_spirit/JsonSpiritHeaders.h>
+
+namespace dev {
+namespace eth {
+namespace validation {
+
+extern std::string const c_sealEngine;
```

Comment:
I liked these constants more in `ChainParams.cpp` / `Account.cpp` ... where they are closer to the actual useful work being done with them.
Are you sure with moving all of this into separate file? I'd leave the three validation functions closer to the other work with the json configs

Reply:
this constants could grow over time into a big amount of variables